packages/consumption/test/modules/notifications/NotificationEnd2End.test.ts

```diff
@@ -66,7 +66,7 @@ describe("End2End Notification via Messages", function () {
         TestNotificationItemProcessor.reset();
     });
 
-    test("sender: sent Notification", async function () {
+    test("sender: save sent Notification", async function () {
```

Comment:
This is a little bit confusing, maybe rephrase: "sender: save Notification with status 'Sent'"